client/checkout/classic/upe.js

```diff
@@ -273,6 +273,7 @@ jQuery( function ( $ ) {
 		// If paying from order, we need to create Payment Intent from order not cart.
 		const isOrderPay = getConfig( 'isOrderPay' );
 		const isCheckout = getConfig( 'isCheckout' );
+		const isChangingPayment = getConfig( 'isChangingPayment' );
```

Comment:
This is a nitpick and not a blocker. 

I noticed this is defined a few times in this file. Is there a reason we can't define this once in the outermost scope of the main function?